### PR TITLE
Reorganize rsub(1) based on patterns used in rinstall(1)

### PR DESCRIPTION
**Pull request by eradman**

### CODE REVIEW

**Comment by vorakl** [comment]
lgtm

File: rsub.sh [hunk 1]
```diff
@@ -12,62 +37,73 @@ usage() {
...
-	case "$arg" in
-		A) append=0 ;;
-		r) line_regex="$OPTARG" ;;
-		l) line_text="$OPTARG" ;;
-		?) usage ;;
-	esac
-done
-shift $(($OPTIND - 1))
-[ $# -eq 1 ] || usage
+	test -f "$target" || {
```
Comment (vorakl) [inline rsub.sh:46]: Just wonder why it is not `[` as in other conditions...

File: rsub.sh [hunk 2]
```diff
@@ -12,62 +37,73 @@ usage() {
...
+	while getopts Al:r: arg; do
+		case "$arg" in
+			A) append=0 ;;
+			r) line_regex="$OPTARG" ;;
+			l) line_text="$OPTARG" ;;
+			?) usage ;;
+		esac
+	done
+	shift $(($OPTIND - 1))
+	[ $# -eq 1 ] || usage
```
Comment (vorakl) [inline rsub.sh:66]: It looks like you used `test` everywhere except this line. Myabe it's worth to also replace it with `test`

File: rsub.sh (hunk 1 above)
Comment (eradman) [reply rsub.sh:46]: Good observation! I'll replace `test` with `[` since this is the pattern in `rinstall`

**Pull request merged**